Makefile

```diff
@@ -75,5 +75,50 @@ check-no-changes:
 	@git diff-index --exit-code HEAD --
 
 .PHONY: test
-test: ## - Run some tests
+test: test-unit #test-int
+
+.PHONY: test-unit
+test-unit: ## - Run some tests
 	@go test -v -race ./...
+
+##################################################
+# Integration testing targets
+##################################################
+
+# Load environment (ES version and creds)
+include ./dev-tools/integration/.env
+export $(shell sed 's/=.*//' ./dev-tools/integration/.env)
+
+# Start ES with docker without waiting
+.PHONY: int-docker-start-async
+int-docker-start-async:
+	@docker-compose -f ./dev-tools/integration/docker-compose.yml --env-file ./dev-tools/integration/.env up  -d --remove-orphans elasticsearch
+
+# Wait for ES to be ready
+.PHONY: int-docker-wait
+int-docker-wait:
+	@./dev-tools/integration/wait-for-elasticsearch.sh ${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD}@localhost:9200
```

Comment:
@mtojek It crossed my mind to potentially use elastic-package here to not require our own yaml compose files etc. Would this make sense?

Reply:
I didn't analyze this use case deeper, but it seems that you're using additional options (ulimits, ES bootstrap, ES discovery). Not sure if you need them or they've been copied from another source. Just in case, these are definitions for elastic-package:  https://github.com/elastic/elastic-package/tree/master/internal/install

Reply:
I copied the settings from my compose files that I used for other projects, some for scale testing, nothing specifically mandatory for our use case, just need the snapshot instance of elasticsearch for integration tests.
Works for now.
Can switch once the PR is merged in the following this PRs or let you adjust how it suppose to be done.

Reply:
Definitively, no urgency to switch over. Was just thinking about simplifying / unifying our test environments long term.